sssom/parsers.py

```diff
@@ -188,24 +197,37 @@ def from_sssom_dataframe(
         df["confidence"].replace(r"^\s*$", np.NaN, regex=True, inplace=True)
 
     mlist: List[Mapping] = []
-    ms = MappingSet()
+    if meta is not None:
+        ms = MappingSet(mapping_set_id=meta["mapping_set_id"], license=meta["license"])
+    else:
+        ms = MappingSet(mapping_set_id=DEFAULT_MAPPING_SET_ID, license=DEFAULT_LICENSE)
     bad_attrs: typing.Counter[str] = Counter()
     for _, row in df.iterrows():
         mdict = {}
         for k, v in row.items():
-            ok = False
-            if k:
-                k = str(k)
-            # if k.endswith('_id'): # TODO: introspect
-            #    v = Entity(id=v)
-            if hasattr(Mapping, k):
-                mdict[k] = v
-                ok = True
-            if hasattr(MappingSet, k):
-                ms[k] = v
-                ok = True
-            if not ok:
-                bad_attrs[k] += 1
+            if v and v == v:
```

Comment:
I have never see this expression. Isnt v==v redundant withjust  v?

Reply:
Or is this to protect agains NaNs?

Reply:
protect against NaNs.